sphinx/setup_command.py

```diff
@@ -144,15 +145,19 @@ def finalize_options(self):
             self.config_dir = self.source_dir
         self.config_dir = abspath(self.config_dir)
 
+        self.ensure_string_list('builder')  # type: ignore
         if self.build_dir is None:
             build = self.get_finalized_command('build')  # type: ignore
             self.build_dir = os.path.join(abspath(build.build_base), 'sphinx')
             self.mkpath(self.build_dir)  # type: ignore
         self.build_dir = abspath(self.build_dir)
         self.doctree_dir = os.path.join(self.build_dir, 'doctrees')
         self.mkpath(self.doctree_dir)  # type: ignore
-        self.builder_target_dir = os.path.join(self.build_dir, self.builder)
-        self.mkpath(self.builder_target_dir)  # type: ignore
+        self.builder_target_dirs = [
+            (builder, os.path.join(self.build_dir, builder))
+            for builder in self.builder]  # type: List[Tuple[str, str]]
```

Comment:
Oh, I'm very sorry. I'd forgotten to say you that you need to import `Tuple` like following:
```
 if False:
     # For type annotation
+    from typing import Any, Tuple  # NOQA
-    from typing import Any  # NOQA
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/206202604#L591